src/plugins/bubble/_base.scss

```
@@ -0,0 +1,145 @@
+
+/*
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * Bubble wizard - Provisional CSS
+ *
+ * ++++++++
+ * This is a Provisional CSS, once stabilized it's include should be done from the theme.scss instead of _provisional.scss
+ * ++++++++
+ *
+ */
+/* Animations */
+
+@keyframes slideInFromRight {
+	0% {
+		transform: scale(0, 1); }
+	95% {
+		transform: scale(0, 1); }
+	100% {
+		transform: scale(1, 1); } }
+
+@keyframes pulseIn {
+	0% {
+		transform: scale(1, 1); }
+	15% {
+		transform: scale(1.15, 1.15); }
+	30% {
+		transform: scale(1, 1); }
+	65% {
+		transform: scale(1.3, 1.3); }
+	100% {
+		transform: scale(1, 1); } }
+
+@keyframes grow {
+	to {
+		transform: translateX(-50%) scale(0); } }
+
+.trans-left {
+	animation-delay: 0s;
+	animation-duration: 5s;
+	animation-iteration-count: 1;
+	animation-name: slideInFromRight;
+	animation-timing-function: ease-out;
+	transform-origin: 100% 50%;
+	will-change: scroll-position; }
+
+.trans-pulse {
+	animation: .5s linear 3.5s 1 pulseIn, .5s linear 15s 1 pulseIn, .5s linear 30s 1 pulseIn;
+	will-change: transform; }
+
+.loader-typing {
+	bottom: 30%;
+	height: 6px;
+	left: 30px;
+	position: absolute;
+	transform: translateX(-50%) translateY(-50%);
+	width: 26px; }
+
+button.wb-sl {
+	background: #26374a;
+	border: none;
+	cursor: pointer;
+	font-weight: 700;
+	padding: 5px;
+	width: auto !important;
+	z-index: 501;}
+
+.wb-bubble-wrap {
+	bottom: 30px;
+	height: 60px;
+	position: fixed;
+	right: 30px;
+	width: 60px;
+	z-index: 1049;
+	p {
+		background: #335075;
+		border-bottom-left-radius: 25px;
+		border-top-left-radius: 25px;
+		box-shadow: 0 1px 3px rgba(0, 0, 0, .45);
+		color: #fff;
+		font-size: 14px;
+		line-height: 20px;
+		min-height: 50px;
+		padding: 5px 37.5px 5px 27.5px;
+		position: relative;
+		right: 195px;
+		top: 5px;
+		width: 225px;
+	}
+	p .notif-close {
+		background: #333;
+		border-radius: 50%;
+		color: #fff;
+		font-size: 19px;
+		height: 1.25em;
+		line-height: 21px;
+		position: absolute;
+		right: 92.5%;
+		text-align: center;
+		text-decoration: none;
+		top: 0;
+		width: 1.25em;
+	}
+	.bubble {
+		background: #fff url("../assets/default-avatar.png") center no-repeat;
+		border-radius: 50%;
+		bottom: 0;
+		box-shadow: 0 2px 4px rgba(0, 0, 0, .45);
+		height: 100%;
+		overflow: hidden;
+		position: absolute;
+		right: 0;
+		text-indent: -9999px;
+		white-space: nowrap;
+		width: 100%;
+		z-index: 1048;
+	}
+	.bubble:focus {
+		border: 1px solid rgba(0, 0, 0, .5);
+		box-shadow: 0 2px 3px rgba(0, 0, 0, .7);
+	}
+}
+
+.wb-bubble-btn-extrnl + .wb-bubble-wrap {
+	display: none !important; }
+
+.wb-bubble-mrgn {
+	margin-top: 80px; }
+
+.wb-bubble-contained {
+	bottom: 0;
+	box-shadow: none;
+	margin: 30px auto;
+	position: static;
+	right: 0;
+	width: 100%; }
+
+.wb-bubble-contained .conversation {
+	max-height: 70vh; }
```

Remove those. They belong to the chat wizard.
```suggestion
.wb-bubble-contained {
	bottom: 0;
	box-shadow: none;
	margin: 30px auto;
	position: static;
	right: 0;
	width: 100%; }

.wb-bubble-contained .conversation {
	max-height: 70vh; }
```